pkgs/applications/editors/emacs/elisp-packages/manual-packages/straight/default.nix

```
@@ -0,0 +1,27 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  lib,
+}:
+
+melpaBuild {
+  pname = "straight";
+  version = "0-unstable-2025-01-25";
```

```suggestion
  version = "0-unstable-2024-10-06";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning